docs/modules/operation/pages/provisioning/detectors/WebDetector.adoc

```diff
@@ -0,0 +1,58 @@
+= Web Detector
+
+Use this detector to find and assign services based on HTTP.
+
+== Detector facts
+
+[options="autowidth"]
+|===
+| Implementation | `org.opennms.netmgt.provision.detector.web.WebDetector`
+|===
+
+== Configuration and Use
+
+.Parameters for the Web detector
+[options="header, autowidth"]
+|===
+| Parameter       | Description                                                                              | Required | Default value
+| `authEnabled`   | Whether to enable basic authentication.                                                  | optional | `false`
+| `authPassword`  | Password that is required for authentication.                                            | optional | `admin`
+| `authPreemtive` | Whether to send basic authentication even if the site did not ask for it.                | optional | `true`
+| `authUser`      | Username that is required for authentication.                                            | optional | `admin`
+| `headers`       | Optional headers to pass in the HTTP request.                                            | optional | -
+| `ipMatch`       | Matches against specific IP addresses or patterns. Filters can be used here to match single addresses or ranges. If there is a ~ at the beginning of the parameter, the rest of the string will be used as a regular expression pattern match, otherwise the match will be a substring match. The regular expression match is anchored at the beginning and end of the line, so you will likely need to put a .* on both sides of your pattern unless you are going to be matching on the entire line.                                      | optional | -
```

Comment:
```suggestion
| `ipMatch`       | Matches against specific IP addresses or patterns. Use filters here to match single addresses or ranges. If there is a ~ at the beginning of the parameter, the rest of the string will be used as a regular expression pattern match, otherwise the match will be a substring one. The regular expression match is anchored at the beginning and end of the line, so you will likely need to put a .* on both sides of your pattern unless you are going to match on the entire line.                                      | optional | -
```